server/migrations/20240415225540_create_users_table.up.sql

```diff
@@ -0,0 +1,13 @@
+-- +migrate Up
+CREATE TABLE users (
+    id SERIAL PRIMARY KEY,
+    username VARCHAR(50) NOT NULL,
```

Comment:
Why are we limiting the length of a username?

Also, `TEXT` is a better default for UTF-8 encoded strings in both Postgres and SQLite